src/backend/postmaster/postmaster.c

```diff
@@ -6326,6 +6329,9 @@ restore_backend_variables(BackendParameters *param, Port *port)
 
 	strlcpy(pkglib_path, param->pkglib_path, MAXPGPATH);
 
+	max_total_memory_mb = param->max_total_memory_mb;
+	max_total_bkend_bytes = (uint64)max_total_memory_mb * 1024 * 1024;
```

Comment:
Surely this is now max_total_memory_bytes..?  Surprised this even compiled..

Reply:
done.